frontend/src/components/Pages/MissionPage/MissionHeader/MissionHeader.tsx

```diff
@@ -9,10 +9,17 @@ import { useLanguageContext, TranslateTextWithContext } from 'components/Context
 import { StatusReason } from '../StatusReason'
 import { MissionRestartButton } from 'components/Pages/FrontPage/MissionOverview/MissionRestartButton'
 
-const HeaderSection = styled.div`
+const HeaderSection = styled(Card)`
+    width: 100%;
+    height: 105px;
     display: flex;
+    top: 60px;
```

Comment:
![image](https://github.com/equinor/flotilla/assets/33518988/f3a60fd5-dab4-4511-861b-04affffbff94)
If header has wrapped content, in looks off when scrolling